plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -621,6 +621,10 @@ private function get_setting_for_feature( string $feature_id, array $feature, bo
 			}
 		}
 
+		if ( 'new_product_management' === $feature_id ) {
+			$desc_tip = __( "⚠ This feature is currently in development and only available for simple products. No extension support yet.", 'woocommerce' );
```

Comment:
@louwie17 I believe we should make it disabled as the HPOS and remove the tooltip for the time being. What do you think?

Reply:
<img width="394" alt="Screenshot 2023-01-23 at 11 49 47" src="https://user-images.githubusercontent.com/13561163/214021776-038e313d-4e9f-4b19-9208-1bfeb98cc12f.png">